project/Build.scala

```diff
@@ -35,7 +35,7 @@ object ExposedValues extends AutoPlugin {
 
 object Build {
 
-  val scalacVersion = "2.11.11" // Do not rename, this is grepped in bin/common.
+  val scalacVersion = "2.12.2" // Do not rename, this is grepped in bin/common.
```

Comment:
Maybe I'm missing something, but this series isn't updating bin/common for 2.12, is it intentional?

Reply:
It wasn't intentional,no. But, we are aware: https://github.com/lampepfl/dotty/pull/3026/

If you think you can fix the `./bin/*` scripts, then we'd be more than happy to accept a PR! :)

Reply:
Great, until now I hadn't even noticed that there is both ./bin and ./dist/bin ;)